litmus/director/TCID-DIR-OP-INSTALL-OPENEBS-CP-ON-SPECIFIC-NODE/test.yml

```diff
@@ -72,6 +73,9 @@
             body_format: json
             status_code: 200
           register: node_details
+          until: "node_details.json.data != []"
+          delay: 2
+          retries: 10
```

Comment:
is this sufficient?

Reply:
I guess so @gprasath. Do you have any thoughts?

Reply:
Keep the delay as it is and increase the retries

Reply:
Ok @gprasath. I'll increase the reries to 30. That should be enough I guess.